upgrade-cockroach-version.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Upgrade a Cluster's Version
+summary: Learn how to upgrade your CockroachDB cluster to a new version.
+toc: false
+toc_not_nested: true
+---
+
+Because of CockroachDB's multi-active availability design, you can perform a "rolling upgrade" of CockroachDB on your cluster. This means you can upgrade individual nodes in your cluster one at a time without any downtime for your cluster.
+
+However, there are a few things you should keep in mind:
+
+  - For rolling upgrades to work, your cluster must be behind a load balancer or your clients must be configured to talk to multiple nodes. If your application communicates with a single node, bringing it down to upgrade its CockroachDB binary will cause your application to fail when the node goes offline.
+
+  - We recommend creating scripts to upgrade CockroachDB and not attempting to do so by hand.
+
+  - Only bring down only one node at a time, and wait at least one minute after it rejoins the cluster to bring down the next node. This reduces the number of nodes you have offline at any point in time, minimizing the chance you'll lose a majority of the nodes in your cluster which can cause service outages.
```

Comment:
Good catch! What I get for trying to finish something before a meeting. Fixed in #1388